dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallKernelMap.java

```diff
@@ -1322,32 +1324,33 @@ public File downloadSingleFeature() {
         data.put(ACTION_ERROR_MESSAGE, null);
         data.put(ACTION_EXCEPTION_STACKTRACE, null);
 
-        ArtifactDownloader artifactDownloader = new ArtifactDownloader();
-        String featureList = (String) data.get(DOWNLOAD_ARTIFACT_LIST);
-        String filetype = (String) data.get(DOWNLOAD_FILETYPE);
+        try (ArtifactDownloader artifactDownloader = new ArtifactDownloader()) {
```

Comment:
Be sure to update the entire call hierarchy for this constructor to use try-with-resources. I only see the one being updated but in Eclipse I can see there are actually 4:
![image](https://user-images.githubusercontent.com/26169178/121252648-425cb480-c876-11eb-8ef7-a376377b11df.png)